index.js

```diff
@@ -80,6 +81,12 @@ const argv = yargs
         type: "boolean",
         default: false
     })
+    .option('heightscale', {

```

Comment:
```suggestion
    .option('heightScale', {

```
To match style of other options. Feel free to use `argv["hs"]` when referencing this later in the code, if that works (I think it does).